ide/src/extension.ts

```diff
@@ -23,6 +26,7 @@ export async function activate(context: vscode.ExtensionContext) {
       ["focus_unmark", focus_unmark],
       ["focus_select", focus_select],
       ["decompose", decompose],
+      ["last_error", last_error.bind(context)],
```

Comment:
A click on a [`StatusBarItem`](https://code.visualstudio.com/api/references/vscode-api#StatusBarItem) is mapped to a vscode command so we need a command which eventually opens the error document. Usually, clicking on the status bar item will toggle focus mode (by calling `flowistry.focus`), but the command switches to `flowistry.last_error` when in an error state.

An alternative is to display the error document when there's both an error in the program and the user is deactivating focus mode. This has the benefit of not creating a new command just for showing the error, but also leads to issues when the user just wants to exit focus mode while the program doesn't compile.